### ISSUE TITLE
Checkable dots in hint mode

### ISSUE THREAD

**Issue by winter-yuki** [braille-systems/learn-braille#270]
Dots are checkable after returning from help.

This code does not help:
```kotlin
if (viewModel.state == DotsChecker.State.HINT) {
    viewModel.expectedDots?.let {
        dotsState.display(it)
    }
}
```

The flip button makes them clickable too.

**Comment by zuevval** [comment]
in which mode/fragment? I cannot reproduce this issue. They are not checkable after returning from `help` in theory and browser

**Comment by zuevval** [comment]
(and in which branch)

**Comment by winter-yuki** [comment]
> in which mode/fragment? I cannot reproduce this issue. They are not checkable after returning from `help` in theory and browser

In practice and in input staps in theory

**Comment by winter-yuki** [comment]
Fixed in #269
